### PR TITLE
Added missing commands, reordered by alpha

### PR DESCRIPTION
**Pull request by larrykooper**
Added `interrupt`, `untracevar`, and `where`. Reordered `delete` and `debug`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,8 +135,10 @@ Command     | Aliases      | Subcommands
 `thread`    |              | `current` `list` `resume` `stop` `switch`
 `tracevar`  |              |
 `undisplay` |              |
+`untracevar`|              |
 `up`        |              |
 `var`       |              | `all` `constant` `global` `instance` `local`
+`where`     |              |
```
Comment (deivid-rodriguez) [inline README.md]: `where` is actually a `backtrace` aliás, which is already listed.

**Comment by larrykooper** [comment]
ok

**Comment by deivid-rodriguez** [comment]
Thank you!

**Comment by larrykooper** [comment]
:)

**Pull request merged**